bindings/rust/wasmedge-sys/Cargo.toml

```diff
@@ -19,8 +19,10 @@ parking_lot = "0.12.1"
 paste = "1.0.5"
 rand = "0.8.4"
 thiserror = "1.0.30"
-wasmedge-macro = {path = "../wasmedge-macro", version = "0.3"}
-wasmedge-types = {path = "../wasmedge-types", version = "0.4"}
+# wasmedge-macro = {path = "../wasmedge-macro", version = "0.3"}
```

Comment:
Do we need to keep these?

Reply:
Accepted. Remove the deprecated ones. Thanks!